dist/screenfull.js

```diff
@@ -23,7 +23,7 @@
 			],
 			// New WebKit
 			[
-				'webkitRequestFullscreen',
+				'webkitEnterFullscreen',
```

Comment:
This won't work as the `webkitEnterFullscreen` only present in the HTMLVideoElement object, not in the `document`.  There should be a way to check whether video object has ability to enter fullscreen.

Reply:
Ye you're right @RusinovAnton, not sure how I missed that. It seems like you can't request fullscreen on the document in Safari? I just tried and nothing worked. Has it always been like this?

Reply:
I couldn't find actual info on that. I can see that there is no `webkitEnterFullscreen` property on `document` object in Safari. However. there is a `document.documentElement.webkitRequestFullscreen` which seems like not working for me. I've tried to find maybe there are some feature flags I can enable, but couldn't find any.

![image](https://user-images.githubusercontent.com/13165540/69038479-8e28de00-09f2-11ea-9519-a1626238169a.png)

Reply:
Uh-oh, it actually works if its triggered not from the console but from the user input even, like button click! - 

`button.addEventListener('click', ()=> document.documentElement.webkitRequestFullscreen())`

Reply:
There is no `document.documentElement.webkitEnterFullscreen` property, tho

and the doc on it is pretty much empty :(

https://developer.apple.com/documentation/tvmljs/document/1630671-documentelement